internal/controller/zz_setup.go

```diff
@@ -148,6 +153,11 @@ func Setup(mgr ctrl.Manager, l logging.Logger, wl workqueue.RateLimiter, ps terr
 		serverpostgresql.Setup,
 		virtualnetworkrule.Setup,
 		providerconfig.Setup,
+		cache.Setup,
+		enterprisecluster.Setup,
+		enterprisedatabase.Setup,
```

Comment:
These do not have custom configs or examples. Do you want to include them?

Reply:
Not sure why they are here, will check.

Reply:
Ok, they are actually coming for `azurerm_redis_enterprise_cluster` and `azurerm_redis_enterprise_database` resources which I missed since they are under redis enterprise section in terraform doc.

Added config and (tested) examples for both.